bigtable/hello_world.rb

```diff
@@ -44,7 +44,7 @@
     column_family,
     column_qualifier,
     value,
-    timestamp: Time.now.to_i * 1000
+    timestamp: (Time.now.to_f * 1_000_000).round { -3 }
```

Comment:
This is probably not what you want. It is returning `-3` from the closure and not passing it as a method argument.

Reply:
What was the lint issue?

Reply:
Oh sorry. I kept getting 2 lint issues that interfered with each other: 

`
bigtable/write_samples.rb:83:48: C: Style/MethodCallWithArgsParentheses: Omit parentheses for method calls with arguments.
  timestamp = (Time.now.to_f * 1_000_000).round(-3)
`

and 

`
bigtable/write_samples.rb:46:49: W: Lint/AmbiguousOperator: Ambiguous negative number operator. Parenthesize the method arguments if it's surely a negative number operator, or add a whitespace to the right of the - if it should be a subtraction.
  timestamp = (Time.now.to_f * 1_000_000).round -3
`

Reply:
Shoot, I guess you might want to suppress one or the other.